console/views/store.py

```diff
@@ -92,18 +92,31 @@ def reprocess_submission():
 @flask_security.login_required
 def store(page):
     audited_logger = logger.bind(user=flask_security.core.current_user.email)
-
     valid = request.args.get('valid', type=str, default='')
     tx_id = request.args.get('tx_id', type=str, default='')
     ru_ref = request.args.get('ru_ref', type=str, default='')
     survey_id = request.args.get('survey_id', type=str, default='')
     datetime_earliest = request.args.get('datetime_earliest', type=str, default='')
     datetime_latest = request.args.get('datetime_latest', type=str, default='')
 
+    # These two variables are either empty, or datetime objects.  A separate variable had
+    # to be used as we need the string representation for the database filter and a
+    # datetime representation of the date for the StoreForm object.
+    datetime_earliest_value = None
+    datetime_latest_value = None
+    if datetime_earliest:
+        datetime_earliest_value = datetime.strptime(datetime_earliest, '%Y-%m-%dT%H:%M')
+
+    if datetime_latest:
+        datetime_latest_value = datetime.strptime(datetime_latest, '%Y-%m-%dT%H:%M')
+
     form = StoreForm(
+        valid = valid,
         tx_id=tx_id,
         ru_ref=ru_ref,
-        survey_id=survey_id
+        survey_id=survey_id,
+        datetime_earliest = datetime_earliest_value,
+        datetime_latest = datetime_latest_value
```

Comment:
These three should have no whitespace... `valid=valid,`

Reply:
Old habits die hard with coding standards :'(  Fixed now!